addons/mail/tests/test_res_users_settings.py

```diff
@@ -1,14 +1,34 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo.addons.mail.tests.common import MailCommon
-from odoo.tests.common import tagged, users
+from odoo.tests.common import tagged, users, TransactionCase
+
+
+class TestUSerCreationSettings(TransactionCase):
+
+    def test_create_portal_user(self):
```

Comment:
Note: most of these tests should be moved in `base` module, it looks like an oversight of the move from `mail` to `base`.

Only those testing mail specific fields should stay here.